### PR TITLE
Use glog instead of fmt

### PR DESCRIPTION
**Pull request by linyouchong**
**What this PR does / why we need it**:
Use glog instead of fmt

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #
NONE
**Special notes for your reviewer**:
NONE
**Release note**:
```release-note
NONE
```

/sig storage

### CODE REVIEW

File: pkg/volume/csi/labelmanager/labelmanager.go [hunk 1]
```diff
@@ -153,7 +153,7 @@ func verifyAndAddNodeId(
 			string(jsonObj))
 		_, updateErr := k8sNodesClient.Update(result)
 		if updateErr == nil {
-			fmt.Printf(
+			glog.V(1).Infof(
```
Comment (msau42) [inline pkg/volume/csi/labelmanager/labelmanager.go]: V(1) seems a little questionable.  @saad-ali says this path should only be called when a driver first registers, and even if the driver crashes and restarts, if it's not recreating it's socket then it should not come here.  Given that, I think V(2) would be better, and we should change all the V(1) to V(2).

**Comment by msau42** [comment]
Can you also fix your release note formatting and replace it with "none"

**Comment by linyouchong** [comment]
@msau42 Fixed, Thanks for your review!

**Pull request merged**